README.md

```diff
@@ -30,9 +30,9 @@ Interested parties can also [join our #security channel](https://communityinvite
 - Joe Sepi ([@joesepi](https://github.com/joesepi))
 - Jordan Harband ([@ljharb](https://github.com/ljharb))
 - Matt Rutkowski ([@mrutkows](https://github.com/mrutkows))
-- Matteo Collina ([@mcollina](https://github.com/mcollina))
```

Comment:
is this intended?

Reply:
Yes? https://github.com/openjs-foundation/security-collab-space/pull/223/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L33 Just kept it in line w/ Chris' PR (minus the other change). I could undo that if we are not going to merge that PR in

Reply:
ah, gotcha. i think it's better to keep this change to that PR, and keep this one to just adding you :-)

Reply:
reverted the change